awx/ui_next/src/screens/InstanceGroup/shared/InstanceGroupForm.jsx

```diff
@@ -11,15 +11,17 @@ import { required, minMaxValue } from '../../../util/validators';
 import { FormColumnLayout } from '../../../components/FormLayout';
 
 function InstanceGroupFormFields({ i18n }) {
+  const [instanceGroupNameField, ,] = useField('name');
   return (
     <>
       <FormField
+        name="name"
         id="instance-group-name"
         label={i18n._(t`Name`)}
-        name="name"
         type="text"
         validate={required(null, i18n)}
         isRequired
+        isDisabled={instanceGroupNameField.value === 'tower'}
```

Comment:
In addition to the validation ranges you've provided, please add a min & max value for the inputs - They're supported and improve the UX when tuning these values with a mouse wheel or keyboard arrows:

![Screenshot from 2020-08-11 09-33-22](https://user-images.githubusercontent.com/9753817/89903821-2173f600-dbb6-11ea-9228-09fe4d8916ac.png)

Reply:
Thanks for your review @jakemcdermott. Updating now.